test/hotspot/jtreg/runtime/Thread/TestAlwaysPreTouchStacks.java

```diff
@@ -167,16 +167,19 @@ public static void main(String[] args) throws Exception {
           if (pretouch_committed == 0 || no_pretouch_committed == 0) {
             throw new RuntimeException("Could not run with PreTouch flag.");
           }
-          long expected_delta = numThreads * (max_stack_usage_with_pretouch - min_stack_usage_with_pretouch);
+          long expected_delta = numThreads * (max_stack_usage_with_pretouch - min_stack_usage_with_pretouch) / 2;
           long actual_delta = pretouch_committed - no_pretouch_committed;
-          if (pretouch_committed <= (no_pretouch_committed + expected_delta)) {
-            throw new RuntimeException("Expected a higher amount of committed with pretouch stacks" +
-                                       "PreTouch amount: " + pretouch_committed +
-                                       "NoPreTouch amount: " + (no_pretouch_committed + expected_delta));
-          }
-          if (actual_delta < expected_delta) {
-            throw new RuntimeException("Expected a higher delta between stack committed of with and without pretouch." +
+          if (pretouch_committed / no_pretouch_committed < 10) {
+            if (pretouch_committed <= (no_pretouch_committed + expected_delta)) {
+              throw new RuntimeException("Expected a higher amount of committed with pretouch stacks" +
+                                       " PreTouch amount: " + pretouch_committed +
+                                       " NoPreTouch amount: " + no_pretouch_committed +
+                                       " Expected Delta calculated as: " + expected_delta);
+            }
+            if (actual_delta < expected_delta) {
+              throw new RuntimeException("Expected a higher delta between stack committed of with and without pretouch." +
                                        "Expected: " + expected_delta + " Actual: " + actual_delta);
+            }
```

Comment:
Is this meant to be part of this PR?

Reply:
Better and correct if not to. Changes are removed.